lib/staccato/V4/event.rb

```diff
@@ -0,0 +1,46 @@
+module Staccato::V4
+  module Event
+    # this module is included into each model hit type
+    #   to share the common behavior required to hit
+    #   the Google Analytics /collect api endpoint
+    def self.included(event)
+      event.extend Forwardable
+
+      event.class_eval do
+        attr_accessor :tracker, :options
+
+        def_delegators :@options, *event::FIELDS.keys
```

Comment:
@tpitale, I think this should just be `event::fields`, because it is an array of symbols instead of an object.